### DISCUSSION TITLE
Vitrotronic 200 Typ KW1

### DISCUSSION THREAD

**Discussion by SvenPausH** [openv/openv#555] [General]
Hallo, funktioniert openv auch mit der Vitrotronic 200 Typ KW1?
Ich habe an der Steuerung 2 LED die eine sieht aus wie ein Blitz dann kommt ein V und rechts eine grüne LED die immer leuchtet.
Wenn ich mit der Handy Cam die linke LED anleuchte sehe ich nichts. Deshalb vermute ich dass das keine Infrarot LED ist.

**Comment by SvenPausH** [comment]
Ist das richtig das die Infrarot LED im V sitzt? 
Dann hätte ich das nicht. Das V ist zwar vertieft, ist aber geschlossen.
Lässt sich die Schnittstelle nachrüsten?

**Comment by speters** [comment]
Die Schnittstelle ist in https://github.com/openv/openv/wiki/Die-Optolink-Schnittstelle beschrieben.

**Comment by SvenPausH** [comment]
Vielen Dank, die rechte LED hat eine Doppelte Funktion. 
Danke dann Löte ich mir das zusammen :).
